scripts/oleani-lib.lic

```diff
@@ -699,7 +699,7 @@ module Oleani
 
       lines = Oleani::Helpers.quiet_command(
           command,
-          /^(?:You are carrying nothing at this time|You are currently (?:wearing and )?carrying)/,
+          /^(?:You are carrying nothing at this time|You are currently (?:wearing)|(?:wearing and carrying))/,
```

Comment:
- You are carrying nothing at this time.
- You are currently wearing
- You are currently holding
- You are holding nothing at this time.

are the potential matches now it looks like for the inv full and inv hands full commands.
`^You are (?:currently )?(?:carrying|holding|wearing)(?: nothing at this time)?`

https://rubular.com/r/mkLWB9l0JE4ESr

should catch them all